rover/source/docs/rover-tuning-process.rst

```diff
@@ -45,12 +45,17 @@ Follow the instructions on the :ref:`Tuning Steering Rate <rover-tuning-steering
 
 3. Set :ref:`ATC_STR_RAT_MAX <ATC_STR_RAT_MAX>` to the :ref:`ACRO_TURN_RATE <ACRO_TURN_RATE>` value and optionally reduce :ref:`ACRO_TURN_RATE <ACRO_TURN_RATE>`
 
+Rover QuickTune LUA Script
+==========================
+
+If the autopilot is capable of running :ref:`common-lua-scripts`, the instead of manually tuning steering and speed/throttle controllers, it ca be done automatically using the `rover-quicktune.lua applet <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/applets/rover-quicktune.lua>`__. Read its `instruction file <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/applets/rover-quicktune.md>`_ for setup and use information.
```

Comment:
thanks for this.

A couple of typos on this line, "the instead" should just be "instead" and "it ca be" should be "it can be".

Txs again